### DISCUSSION TITLE
Easier imports

### DISCUSSION THREAD

**Discussion by thewh1teagle** [brython-dev/brython#2450] [Q&A]
Currently when importing other `python` files, I have to add the following to the `html` file:

```html
<script src="/common/first.py" type="text/python" id="first"></script>
<script src="/common/second.py" type="text/python" id="second"></script>
<body> 
  <script type="text/python" src="./index.py"></script> 
</body>
```
In `index.py` I can then import like the following:

```python
import first
import second
```

Is there way to import them without declare them inside `index.html` so I can easily add more files and import?

**Comment by PierreQuentel** [comment]
Like in CPython, you can just put the modules **`first.py`** and **`second.py`** in the main script folder, or in a directory with an **`__init__.py`**
